src/pages/home/HeaderView.js

```diff
@@ -100,11 +101,15 @@ const HeaderView = (props) => {
                             {props.report.hasOutstandingIOU && (
                                 <IOUBadge iouReportID={props.report.iouReportID} />
                             )}
+                            <VideoChatButtonAndMenu />
                             <Pressable
                                 onPress={() => togglePinnedState(props.report)}
                                 style={[styles.touchableButtonImage, styles.mr0]}
                             >
-                                <Icon src={Pin} fill={props.report.isPinned ? themeColors.heading : themeColors.icon} />
+                                <Icon
```

Comment:
This change can be reverted

Reply:
I'm gonna keep this one bc it's giving the line length warning (over 120)

Reply:
Alright, not seeing that on my end but not a big deal

Reply:
Oh heh, I see what happened here. The line is _exactly_ 120, and when you put a trailing space at the end, it goes over. Sometimes to get the linter to run on a given file I put a trailing space in so that it re-runs. Reverting 😅